rpa-manager/modules/ROOT/pages/_partials/form-scheduling.adoc

```diff
@@ -21,33 +21,33 @@ Then, under *Repeats*, you can choose whether the *From-To* time interval is to
 +
 The duration of the contained time interval and the start time are taken into account. The duration of the time interval is defined as the difference between the *From* and *To* times. The *From* time is the start time of the first execution. The start time of repetitions depends on the other set values.
 +
-If you wish to execute the repeats only with reference to the time of day, you should ensure that the start and end date are identical. If the interval specified  covers multiple days, a new multi-day interval begins with every repetition. This can lead to a situation whereby the scheduled time interval lasts longer than expected because the full number of days contained in the *From-To* interval is executed once again during the last repetition.
+If you want to execute the repeats only with reference to the time of day, you should ensure that the start and end date are identical. If the interval specified  covers multiple days, a new multi-day interval begins with every repetition. This can lead to a situation whereby the scheduled time interval lasts longer than expected because the full number of days contained in the *From-To* interval is executed once again during the last repetition.
```

Comment:
```suggestion
To execute the repeats only with reference to the time of day, ensure that the start and end dates are identical. If the interval specified covers multiple days, a new multi-day interval begins with every repetition. This can lead to a situation in which the scheduled time interval lasts longer than expected because the full number of days contained in the *From-To* interval is executed once again during the last repetition.
```

Reply:
Suggest: To execute the repeats referencing only the time of day